### PR TITLE
[CB-173] Clean up codebase, formatting and indentation

### PR DESCRIPTION
**Pull request by LWanjiru**
# JIRA TICKET NAME:
CB-173 - [Clean up codebase, formatting and indentation](https://issues.openmrs.org/browse/CB-173)

## SUMMARY:
Currently, some part of the codebase is not properly formatted or indented. 

This task is will ensure that the codebase is properly formatted and fix all issues as reported by HoundCi

### CODE REVIEW

**Comment by joel-ace** [comment]
There are still some issues raised by HoundCI

File: .eslintrc [hunk 1]
```diff
@@ -32,19 +30,16 @@
     "no-var": 1,
     "import/no-unresolved": 0,
     "semi": [1, "always"],
-    "no-trailing-spaces": 0,
```
Comment (cnagadya) [inline .eslintrc]: Any reason why this & eol-last were removed?

**Review by malmike — COMMENTED** [review]
Still some issues raised by HoundCI

File: .eslintrc (hunk 1 above)
Comment (LWanjiru) [reply .eslintrc]: Because it was saying not to give any warnings, so essentially it was allowing trailing spaces

**Comment by LWanjiru** [comment]
@malmike @joel-ace this is because I changed the settings of the .eslintrc file and hound is using the old settings stored in the master branch.

**Comment by LWanjiru** [comment]
All HoundCI issues have now been resolved.

File: app/js/components/tabs/tabcomponents/savedComponent.jsx [hunk 2]
```diff
@@ -431,9 +426,7 @@ class SavedComponent extends React.Component {
           downloadJobs={this.state.cohortDownloadJobs}
           results={this.state.cohortResults}
           isSearching={this.state.searchingCohorts}
-          tableName={this.state.inSearchCohortMode ?
-            'Cohort Search Results':
-            ''}
+          tableName={this.state.inSearchCohortMode ? 'Cohort Search Results' : ''}
```
[the remark this answers is not in the document]
Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/savedComponent.jsx:429]: Resolved.

File: app/js/components/tabs/tabcomponents/savedComponent.jsx [hunk 3]
```diff
@@ -383,45 +380,43 @@ class SavedComponent extends React.Component {
...
+                className="form-control"
+                placeholder="Enter Cohort Name . . ." 
+                onChange={this.handleInputChange}
+              />
+              <span
+                className="input-group-btn">
+                <button
+                  className="btn btn-success"
+                  disabled={this.state.searchingCohorts ? 'disabled' : null}
+                  onClick={this.searchSavedCohorts}
```
[the remark this answers is not in the document]
Comment (LWanjiru) [reply app/js/components/tabs/tabcomponents/savedComponent.jsx:413]: Resolved.

**Pull request merged**